capsules/extra/src/hs3003.rs

```diff
@@ -186,7 +186,10 @@ impl<'a, I: I2CDevice> I2CClient for Hs3003<'a, I> {
                     self.pending_humidity.set(false);
                     self.humidity_client.map(|client| client.callback(humidity));
                 }
+
+                self.state.set(State::Sleep(temperature, humidity));
```

Comment:
Do we need the two values with the `Sleep` state or can those be removed?

Reply:
Your right, there is no need for the `Sleep` state to have values at all since I removed the last `Sleep` state.